sdcclient/_monitor.py

```diff
@@ -754,6 +748,52 @@ def get_metrics(self):
         res = requests.get(self.url + '/api/data/metrics', headers=self.hdrs, verify=self.ssl_verify)
         return self._request_result(res)
 
+    def convert_scope_string_to_expression(self, scope):
+        if scope != None:
+            expressions = []
+            string_expressions = scope.strip(' \t\n\r').split(' and ')
+            expression_re = re.compile('^(?P<not>not )?(?P<operand>[^ ]+) (?P<operator>=|!=|in|contains|starts with) (?P<value>.+)$')
+
+            for string_expression in string_expressions:
+                matches = expression_re.match(string_expression)
+
+                if matches == None:
+                    return [False, 'invalid scope format']
+
+                is_not = matches.group('not') != None
```

Comment:
PEP8 recommendations say we should use `is None` and `is not None` instead of `== None` and `!= None`. This is happening many times at the PR, so I guess you are using `==` to maintain code homogeneity, in that case :+1:

Reply:
Are they functionally equivalent? Also, this reminds me that we should really add "linting" validation soon...